### PR TITLE
Placeholders translated on register form

### PR DESCRIPTION
**Pull request by fucheron**

### CODE REVIEW

File: src/localization/en.js [hunk 1]
```diff
@@ -112,6 +112,8 @@ export default {
     }
   },
   terms: {
+    beforeContent:'Accept the',
```
Comment (florenperetti) [inline src/localization/en.js:115]: change to "I accept the "

Comment (florenperetti) [inline src/localization/en.js]: Change to "I accept the "

File: src/views/RegisterView.vue [hunk 2]
```diff
@@ -46,16 +46,15 @@
...
+                                    name="mail"
+                                    :placeholder="$t('register.placeholders.email')" />
                             </div>
                             <div class="form-input">
                                 <input
                                     type="password"
                                     v-model="user.password"
-                                    :name="$t('register.placeholders.password')"
-                                    placeholder="Contraseña" />
+                                    name="mail"
```
Comment (florenperetti) [inline src/views/RegisterView.vue]: Fix: should be name="password"

**Review by florenperetti — CHANGES_REQUESTED** [review]
@osgigon some little fixes here

**Review by florenperetti — APPROVED** [review]

**Review by florenperetti — APPROVED** [review]

**Review by florenperetti — APPROVED** [review]

**Review by CristianPeralta — APPROVED** [review]

**Pull request merged**